build/Toolset.proj

```diff
@@ -10,12 +10,9 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="RoslynTools.RepoToolset" />
-    <PackageReference Include="RoslynTools.VsixExpInstaller" />
     <PackageReference Include="Codecov" />
     <PackageReference Include="OpenCover" />
     <PackageReference Include="Microsoft.DiaSymReader.Pdb2Pdb" />
-    <PackageReference Include="Microsoft.DevDiv.Validation.MediaRecorder" />
```

Comment:
Note: without these you will not have any screenshots or videos output on CI.  I have no idea how you will debug CI in these cases

Reply:
See the temp artifacts directory on this [build](https://dev.azure.com/dnceng/public/_build/results?buildId=274165)

Reply:
These were removed previously - see https://github.com/dotnet/project-system/issues/5092 that is tracking readding them.

This is what we found with both of these existing ones:

1) Screenshots are already taken by Apex or one of its installed plugins. The screen shot taken by the above collector never actually captured the screen at the failure.
2) We couldn't actually get media recorder to work at all, we plan in debugging why it wasn't working and readding it.

Reply:
Thanks, I do see a screen recording, could never getting that working will retry tomorrow. 

In that folder you'll see some screenshots with "Fail.[randomchars].png" that is the built-in/already installed screenshot take, the OnTestCaseFail.png is the one above. We felt it added nothing (in fact in some cases worse) then the other one so we've removed it.

Reply:
alright, well it was recording video as recently as that build I pointed out

Reply:
Does screen recording happen on your box? It doesn't on mine - I wonder if its machine specific, I did check and it seems to bail on dual screens - so wonder if bails on other cases.

Reply:
I am perfectly alright with not having the additional screen shots